### ISSUE TITLE
[Bug]  Error initializing vtkGenericRenderWindow API in vtk.js@29.0.0 version

### ISSUE THREAD

**Issue by zhangzhi93** [Kitware/vtk-js#2952]
### Bug description

vtk.js@29.0.0 version of the vtkGenericRenderWindow API initialization error, the error message is Cannot read properties of undefined (reading'setRenderable ').

```javascript
  publicAPI.addView = view => {
    if (publicAPI.hasView(view)) {
      return;
    }
    view.setRenderable(publicAPI);
    model._views.push(view);
    publicAPI.modified();
  };
```
View source found that RenderWindow call newAPISpecificView did not run registerViewConstructor registered

### Steps to reproduce

Looking directly at the official example, you will find that it is not rendered correctly, and there is an error message in the console:

https://kitware.github.io/vtk-js/examples/ResliceCursorWidget.html

### Expected Behavior

vtkGenericRenderWindow can render correctly

### Environment

- vtk.js version:29.0.0
- Browsers: chrome 118.0.5993.71
- OS:windows 11

**Comment by MountainAndMorning** [comment]
I encounter the same problem. You can temporarily bypass this problem by replacing the `GenericRenderWindow` with `FullScreenRenderWindow`.

**Comment by zhangzhi93** [comment]
> I encounter the same problem. You can temporarily bypass this problem by replacing the `GenericRenderWindow` with `FullScreenRenderWindow`.

Yes, `FullScreenRenderWindow` won't report an error, but I want to render in a `div`, and I don't want to display it in full screen. I don't know how to configure it.
